### PR TITLE
feat: added collcart website repo (merchandise based)

### PR DESCRIPTION
**Pull request by iamayushdas**

### CODE REVIEW

File: projects/iamayushdas/cart.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+description: Explore new and trendy getups & webseries with Collcart!!
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - HTML / CSS
+  - React
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Data Visualization
+  - Education
+  - Social Activism
```
Comment (ramonh) [inline projects/iamayushdas/cart.mdx:13]: Could you please elaborate on how collcart is for Social Activism?

File: projects/iamayushdas/cart.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+name: Collcart
+repoUrl: https://github.com/iamayushdas/cart.git
+description: Explore new and trendy getups & webseries with Collcart!!
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - HTML / CSS
+  - React
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Data Visualization
```
Comment (ramonh) [inline projects/iamayushdas/cart.mdx:11]: Could you please elaborate on how collcart is for Data Visualization?

File: projects/iamayushdas/cart.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - Education
+  - Social Activism
+  - Web Framework
+  - Merchandise
```
Comment (ramonh) [inline projects/iamayushdas/cart.mdx:15]:
```suggestion
  - E-commerce
```

What do you think about using the tag E-commerce instead?

File: projects/iamayushdas/cart.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    url: https://www.gatsbyjs.com/
+  - title: Get started with Markdown
+    url: https://www.markdownguide.org/getting-started/
+---
+
+<!-- Divide your content into sections using our handy shortcodes. Each section is optional -->
+
+<Overview>
+
+#Its a merchandise for college societies plus college courses are also available on this.
```
Comment (ramonh) [inline projects/iamayushdas/cart.mdx:48]:
```suggestion
# It's a merchandise for college societies plus college courses are also available on this.
```
Is it possible you meant for this to be a header?

**Review by ramonh — CHANGES_REQUESTED** [review]
Thank you so much for adding `collcart`, @iamayushdas!

I've made some questions and change requests.

Could you please also elaborate on some of the overview of the project as well as the contribution guidelines? Please see the template and other projects for ideas.

Thank you in advance!

File: projects/iamayushdas/cart.mdx (hunk 4 above)
Comment (iamayushdas) [reply projects/iamayushdas/cart.mdx]: resolved

File: projects/iamayushdas/cart.mdx (hunk 3 above)
Comment (iamayushdas) [reply projects/iamayushdas/cart.mdx]: resolved

File: projects/iamayushdas/cart.mdx (hunk 2 above)
Comment (iamayushdas) [reply projects/iamayushdas/cart.mdx]: resolved

File: projects/iamayushdas/cart.mdx (hunk 1 above)
Comment (iamayushdas) [reply projects/iamayushdas/cart.mdx]: resolved

**Review by iamayushdas — COMMENTED** [review]
@ramonh have a look now

**Comment by ramonh** [comment]
Looks great, thank you so much, @iamayushdas

**Review by ramonh — APPROVED** [review]

**Pull request merged**